src/itx.rs

```diff
@@ -137,3 +139,277 @@ pub unsafe extern "C" fn inv_txfm_add_rust<BD: BitDepth>(
         dst = dst.offset(BD::pxstride(stride) as isize);
     }
 }
+
+pub type itxfm_fn = Option<
```

Comment:
We can remove the `Option`, right?  We've been doing that for other ones already.

Reply:
Oh, I wanted to avoid doing that in this PR and instead do it when the init fns get cleaned up. I wanted to avoid having to update all of the init logic manually since that's hundreds and hundreds of lines that are all going to get rewritten with a macro down the line most likely. So I figured I'd avoid doing that here to keep this PR simpler.